valoStatus/Region.py

```diff
@@ -42,8 +42,15 @@ def __init__(self, region):
         self.region = region
         
     def requests(self):
-        r = requests.get(f"https://valorant.secure.dyn.riotcdn.net/channels/public/x/status/{self.region.lower()}.json")
-        if r.status_code == 200:
+        url = f"https://valorant.secure.dyn.riotcdn.net/channels/public/x/status/{self.region.lower()}.json"
+        async def requests():
+            async with aiohttp.ClientSession() as session:
+                async with session.options(url) as response:
+                    global STATUS
+                    STATUS = response.status
+        loop = asyncio.get_event_loop()
+        loop.run_until_complete(requests())
+        if STATUS == 200:
             json_data = r.json()
             return json_data
```

Comment:
Issues Found
1. There is no variable with `r` -> "NameError: name 'r' is not defined"
2. You cannot use `r.json()` since its outside of the async function when i put it inside the async function it throws error

Reply:
...yeah, sorry.